### DISCUSSION TITLE
python env and os env

### DISCUSSION THREAD

**Discussion by zhaochenfeng** [apache/dolphinscheduler#10253] [Q&A]
When configuring the environment, the Python environment is not in effect, and the system default environment is used instead
Related configurations have been checked:
![image](https://user-images.githubusercontent.com/95221641/170444412-406172f2-8291-43ff-84b1-b62ac3f23b03.png)
![image](https://user-images.githubusercontent.com/95221641/170444643-eab6f8cb-03ef-48b3-bb9c-aa0c343f76a6.png)
![image](https://user-images.githubusercontent.com/95221641/170444834-019d8d82-3d82-44cb-ab5a-0d970db220e4.png)
Dolphin startup user python are :
![image](https://user-images.githubusercontent.com/95221641/170445542-fd968630-eb2d-431c-b052-837d147107ef.png)
The python in the system usr/bin directory are:
![image](https://user-images.githubusercontent.com/95221641/170445672-6cef436f-a09a-4990-ad79-ceb759237950.png)
But I have specified PYTHON_HOME and other paths, why the printed version is still the system version during the test
![image](https://user-images.githubusercontent.com/95221641/170446110-fcb1e482-7f54-4fe9-8b81-ca97b10bff4b.png)
I would appreciate your help!

**Comment by davidzollo** [comment]
![image](https://user-images.githubusercontent.com/15833811/171453224-6e3dd934-9f5b-4b93-83b8-8be37d5c760c.png)

is this file /etc/profile or dolphinscheduler-env.sh?

**Reply by zhaochenfeng** [reply]
is dolphinscheduler-env.sh

**Reply by weile321** [reply]
@davidzollo I have the same problem,
